File: wrappers/mvc/demos/Kendo.Mvc.Examples/Models/InternetUsers.cs
using System;
using System.Linq;

namespace Kendo.Mvc.Examples.Models
{
    public class InternetUsers
    {
        public InternetUsers()
        {
        }

        public InternetUsers(int year, string country, double value)
        {
            Year = year;
            Value = value;
            Country = country;
        }

        public int Year { get; set; }
        public double Value { get; set; }
        public string Country { get; set; }
    }
}

File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/models/InternetUsers.java
package com.kendoui.spring.models;

public class InternetUsers {
    private int year;
    private double value;
    private String country;
    
    public InternetUsers(int year, double value, String country) {
        setYear(year);
        setValue(value);
        setCountry(country);
    }
    
    public int getYear() {
        return year;
    }
    
    public void setYear(int year) {
        this.year = year;
    }
    
    public double getValue() {
        return value;
    }
    
    public void setValue(double value) {
        this.value = value;
    }
    
    public String getCountry() {
        return country;
    }
    
    public void setCountry(String country) {
        this.country = country;
    }
}
